observability/network_observability/network-observability-secondary-networks.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-secondary-networks"]
+= Secondary networks
+include::_attributes/common-attributes.adoc[]
+:context: secondary_networks
+
+toc::[]
+You can configure the Network Observability Operator to collect and enrich network flow data from secondary networks, such as SR-IOV and Open Virtual Network (OVN)-Kubernetes.
```

Comment:
Is it written like this elsewhere in the docs right now? "Open Virtual Network (OVN)-Kubernetes"

It looks a little odd to me. I did a quick search and didn't see it like that at least in places like https://docs.openshift.com/container-platform/4.17/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html

Could you just call it OVN-Kubernetes here, since it's just an example?

```suggestion
You can configure the Network Observability Operator to collect and enrich network flow data from secondary networks, such as SR-IOV and OVN-Kubernetes.
```

Reply:
I've seen it written like this in some places. I didn't know if I should spell it out completely since its the first mention of it in the NetObserv docs, and in this particular doc. But I feel that normally I hear it as OVN-Kubernetes.